materials/demo.py

```diff
@@ -0,0 +1,39 @@
+"""Demonstration."""
+import numpy as np
+import matplotlib.pyplot as plt
+import material
+
+def main():
+    # pylint: disable=no-member
+    filename = '../materials_data/Al_6061.yaml'
+    al6061 = material.load_from_yaml(filename, 'extruded, thickness > 1 inch', 'T6')
+
+    temperature = np.linspace(80, 580)
```

Comment:
If I put my matt-hat on, I might ask for units

Reply:
https://github.com/mvernacc/material-properties-interchange/pull/1/commits/8231cb717171b9b22b75e592594a69c1ac5f5361